contracts/libraries/PlasmaVaultLib.sol

```diff
@@ -105,10 +284,38 @@ library PlasmaVaultLib {
         return PlasmaVaultStorageLib.getManagementFeeData();
     }
 
-    /// @notice Configures the management fee data like the fee manager and the fee in percentage
-    /// @param feeAccount_ The address of the technical Management Fee Account that will receive the management fee collected by the Plasma Vault and later on distributed to IPOR DAO and recipients by FeeManager
-    /// @param feeInPercentage_ The fee in percentage, represented in 2 decimals, example: 100% = 10000, 1% = 100, 0.01% = 1
-    /// @dev feeAccount_ can be also EOA address or MultiSig address, in this case it will receive the management fee directly
+    /// @notice Configures the management fee settings for the vault
+    /// @param feeAccount_ The address that will receive management fees
+    /// @param feeInPercentage_ The management fee rate in basis points (100 = 1%)
+    /// @dev Updates fee configuration and emits event
+    ///
+    /// Parameter requirements:
+    /// - feeAccount_: Must be non-zero address
```

Comment:
The address of the technical Management Fee Account that will receive the management fee collected by the Plasma Vault and later on distributed to IPOR DAO and recipients by FeeManager